keyboards/kinesis/config.h

```diff
@@ -136,5 +136,8 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #ifdef SUBPROJECT_stapelberg
     #include "stapelberg/config.h"
 #endif
+#ifdef SUBPROJECT_kint2pp
```

Comment:
I don't believe that these are used anymore, actually.

Reply:
Yeah, Lines 133-141 aren't needed here at all, and deleting makes no difference to the build result.